docs/source/concepts/cl_columns.rst

```diff
@@ -21,37 +28,95 @@ The Cleanlab columns are listed below in the table. Detailed explanations of eac
 
    * - Cleanlab Column Name
```

Comment:
Can sort/group these by what they apply to, e.g., group label-related ones together